docs/src/configuration/services/memcached.md

```diff
@@ -26,16 +26,16 @@ The format exposed in the ``$PLATFORM_RELATIONSHIPS`` [environment variable](/de
 
 ## Usage example
 
-In your ``.platform/services.yaml``:
+{{% endpoint-description type="memcached" %}}
+
+Service definition:
 
 {{< readFile file="src/registry/images/examples/full/memcached.services.yaml" highlight="yaml" >}}
 
-Now add a relationship in your `.platform.app.yaml` file:
+App configuration:
 
 {{< readFile file="src/registry/images/examples/full/memcached.app.yaml" highlight="yaml" >}}
```

Comment:
can we add the .platform.app.yaml OR applications.yaml tab to all of these examples? It'll be closer to the descriptive version we had before, which sets the right context on where it needs to be added 🤔

Reply:
I'd link to the appropriate page describing the configuration in detail. Having both names seems complicated. The link offers the context to those who might want it while not getting in the way of people who don't.

Reply:
Right. I'd lean towards one file name. But yeah, this would impact all services pages and requires the alignment we were talking about in another comment.

Reply:
Let's resolve this one now that we have agreed on how to approach this?

Reply:
@MariadeAnton , hope I just added it everywhere relevant.